CleverTapSDK/CleverTap.m

```diff
@@ -2281,9 +2320,18 @@ - (void)sendQueue:(NSMutableArray *)queue ofType:(CTQueueType)queueType {
 
 #pragma mark Response Handling
 
-- (void)parseResponse:(NSData *)responseData {
+- (void)parseResponse:(NSData *)responseData responseEncrypted:(BOOL)responseEncrypted {
     if (responseData) {
         @try {
+            if (responseEncrypted) {
+                if (@available(iOS 13.0, *)) {
```

Comment:
If the response is encrypted but the iOS is < 13.0, shouldn't this method just return to exit? I assume this should not happen in any case since the request will be encrypted only on iOS 13.0 and above, however, I think we can guard against it here, since the check is done anyway.

Reply:
done